qcengine/programs/nwchem/keywords.py

```diff
@@ -79,6 +79,8 @@ def rec_dd():
                     lines.append(line)
         if group == "aaaglobal":
             grouped_lines[group] = "\n".join(lines) + "\n"
+        elif group.lower() == "set":
+            grouped_lines[group] = "\n".join(f"set {l}" for l in lines) + "\n"
```

Comment:
what is the "set" addition doing? I'm not tracing it.

Reply:
Instead of using the "block\n....\end" syntax to house multiple options of the same group, it writes out each "set" option as its own line.

E.g., 

Produces
```
set opt_1 value1
set opt_2 value2
```

instead of 

```
set
  opt_1 value1
  opt_2 value2
end
```

Reply:
is this for top-level/unscoped keywords? so scoped becomes `set ccsd opt_1 value1`?

Reply:
I'm a little wary of this in case it interferes with qcdb options handling. but ok for now. thanks for showing me new nwc syntax :-)